tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs

```diff
@@ -48,7 +50,9 @@ public static class Program
         string codeownersFilePathOrUrl,
         bool excludeNonUserAliases = false,
         string? targetDir = null,
-        string ignoredPathPrefixes = DefaultIgnoredPrefixes)
+        string ignoredPathPrefixes = DefaultIgnoredPrefixes,
+        string? teamStorageURI = null,
+        string? ownersDataOutputFile = null)
```

Comment:
@weshaggard, I had to add another command line option to RetrieveCodeOwners to allow writing the json output to the file. The fact is, the [get-codeowners.lib.ps1](https://github.com/Azure/azure-sdk-tools/blob/952b6db75fd998adc6cf83af03da3a158ab1ee80/eng/common/scripts/get-codeowners.lib.ps1#L125) was incorrectly assuming any/all output from the executable was nothing more than a json codeowner entry. Using CodeOwnersParser FileHelper [GetUrlContents](https://github.com/Azure/azure-sdk-tools/blob/main/tools/code-owners-parser/CodeOwnersParser/FileHelpers.cs#L38) to fetch the blob, will cause output that'll break this assumption. The correct fix for this is the add the option to write the entry to a file. The problem here is because of locked version in the [eng/common/get-codeowners.lib.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/get-codeowners.lib.ps1#L81) script, I can't make the changes, to use the file instead of output, to the script until this is checked in and another version has been published.
@konrad-jamrozik and I already chatted about this and both agreed this was the correct way to fix this.